docs/development-environment.md

```diff
@@ -55,6 +55,13 @@ $ pip install -r requirements/develop.pip
 $ pip install -e .
 ```
 
+Also, we're using [pip-tools](https://github.com/jazzband/pip-tools/#pip-tools--pip-compile--pip-sync) with [a pre-commit hook](https://github.com/jazzband/pip-tools/#version-control-integration) to help us manage our requirements. **This is not mandatory unless you're actively contributing to the project.** `pip-tools` uses the `.in` files in `requirements/` as input to generate the `.pip` files we rely on to install `udata`. If you need to add or modify a dependency, do it in the `.in` files. The pre-commit hook will compile the `.pip` files and warn you.
```

Comment:
Is it recommended to rely on `pre-commit` to update `.pip` files or could we use `pip-compile` directly?

Reply:
We could but this leads to a long list of commands to get the right files in and out. The advantage of the pre-commit hook is that everything is stored in the conf file (and we can change the conf file w/o updating the doc :p).

Reply:
(Since I've started working w/ pip-tools, I find it easier to rely on the pre-commit hook)

Reply:
Done!